File: cs/helloworld.cs
using System;

public class HelloWorld
{
  static public void Main ()
  {
    Console.WriteLine ("Hello World!");
    int x = 12 * 30;
    Console.WriteLine (x);
    Console.WriteLine (Double(x));
  }

  static int Double (int x)
  {
    int doubled = x * 2;
    return doubled;
  }

}

File: java/University_Projects/jsftest/src/main/java/com/ryanfleck/jsftest/HelloWorld.java
package com.ryanfleck.jsftest;

import javax.faces.bean.ManagedBean;

// https://www.tutorialspoint.com/jsf/jsf_managed_beans.htm

// Eager -> Bean created before it is requested.
@ManagedBean(name = "helloWorld", eager = true)
public class HelloWorld {

	public HelloWorld() {
		System.out.println("HelloWorld started!");
	}

	public String getMessage() {
		return "Hello World!";
	}

}
